at_client/lib/src/responseParser/notification_response_parser.dart

```diff
@@ -0,0 +1,12 @@
+class ResponseParser {
```

Comment:
Sita, there is a standard ResponseParser interface. Naresh wrote a few and it is in a package. Please pull all of the here.